src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -46,21 +47,45 @@
         <Grid x:Name="LeftGrid"
               Grid.Row="1"
               Grid.Column="0">
-            <graphControl:Grapher Name="GraphingControl"
-                                  Margin="4,7,4,4"
+            <Grid Grid.Row="0" Margin="4,7,4,4">
+                <graphControl:Grapher Name="GraphingControl" 
                                   EquationsSource="{x:Bind ViewModel.Equations, Mode=OneWay}"
                                   ForceProportionalAxes="True"
                                   UseSystemFocusVisuals="True"
-                                  VariablesUpdated="GraphVariablesUpdated">
-                <graphControl:Grapher.Background>
-                    <SolidColorBrush Color="White"/>
-                </graphControl:Grapher.Background>
-                <graphControl:Grapher.EquationTemplate>
-                    <DataTemplate x:DataType="vm:EquationViewModel">
-                        <graphControl:Equation Expression="{x:Bind Expression, Mode=OneWay}" LineColor="{x:Bind LineColor, Mode=OneWay}"/>
-                    </DataTemplate>
-                </graphControl:Grapher.EquationTemplate>
-            </graphControl:Grapher>
+                                  VariablesUpdated="GraphVariablesUpdated"
+                                  LostFocus="OnGraphLostFocus"
+                                  LosingFocus="OnLoosingFocus">
+                    <graphControl:Grapher.Background>
+                        <SolidColorBrush Color="White"/>
+                    </graphControl:Grapher.Background>
+                    <graphControl:Grapher.EquationTemplate>
+                        <DataTemplate x:DataType="vm:EquationViewModel">
+                            <graphControl:Equation Expression="{x:Bind Expression, Mode=OneWay}" LineColor="{x:Bind LineColor, Mode=OneWay}"/>
+                        </DataTemplate>
+                    </graphControl:Grapher.EquationTemplate>
+
+                </graphControl:Grapher>
+
+                <StackPanel Grid.Row="0" Orientation="Horizontal" HorizontalAlignment="Right" VerticalAlignment="Top" Margin="0,0,10,10">
+                    <Button x:Name="ActiveTracing" Click="OnActiveTracingClick" MinWidth="40" MinHeight="40" Margin="0,0,10,0">
+                        <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE73F;"/>
```

Comment:
Since this is a FontIcon now can we delete this file?
src/Calculator/Assets/ActiveTracking-32_contrast-white.png

Reply:
Yep, I had removed it from the solution but it didn't cleanup the file system.